File: leetcode/easy/3069-distribute-elements-into-two-arrays-i/3069-distribute-elements-into-two-arrays-i.kt
class Solution {
    fun resultArray(nums: IntArray): IntArray {
        val arr1 = mutableListOf<Int>()
        val arr2 = mutableListOf<Int>()

        arr1.add(nums[0])
        arr2.add(nums[1])

        for (i in 2 until nums.size) {
            if (arr1.last() > arr2.last()) {
                arr1.add(nums[i])
            } else {
                arr2.add(nums[i])
            }
        }

        return (arr1 + arr2).toIntArray()
    }
}

File: leetcode/easy/3069-distribute-elements-into-two-arrays-i/3069-distribute-elements-into-two-arrays-i.py
# Definition for singly-linked list.
from typing import List


class Solution:
    def resultArray(self, nums: List[int]) -> List[int]:
        arr1 = []
        arr2 = []

        arr1.append(nums.pop(0))
        arr2.append(nums.pop(0))

        while nums:
            if arr1[-1] > arr2[-1]:
                arr1.append(nums.pop(0))
            else:
                arr2.append(nums.pop(0))

        return arr1 + arr2
